src/docs/asciidoc/web.adoc

```diff
@@ -1,25 +1,30 @@
 [[spring-web]]
-= Web
+= The Web
 :doc-root: https://docs.spring.io
 :api-spring-framework: {doc-root}/spring-framework/docs/{spring-version}/javadoc-api/org/springframework
 :toc: left
 :toclevels: 2
 
-This part of the documentation covers support for web applications designed to run on a
-traditional Servlet stack (Servlet API + Servlet container).
```

Comment:
As stated previously, this document now concerns only the servlet-based stack, so this statement is on purpose.